### PR TITLE
Document testing local builds and remove VS version

### PR DESCRIPTION
**Pull request by drewnoakes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
-## Build Status
+## Build status
 
 |Windows x64 |
 |:------:|
 |[![](https://dev.azure.com/dnceng/internal/_apis/build/status/dotnet/sdk/DotNet-Core-Sdk%203.0%20(Windows)%20(YAML)%20(Official))](https://dev.azure.com/dnceng/internal/_build?definitionId=140)|
 
+## Testing a local build
+
+To test your locally built SDK, run `eng\dogfood.cmd` after a building. That script starts a new Powershell with the environment configured to redirect SDK resolution to your build. Then run `& devenv.exe`.
```
Comment (nguerrera) [inline README.md]: We should also doc that msbuild and dotnet build will use your changes as well.

**Review by nguerrera — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (drewnoakes) [reply README.md]: Outside of VS you mean?

Comment (nguerrera) [reply README.md]: Yeah, I mean you can do command line testing of your bits this way.

Comment (nguerrera) [inline README.md]:
```suggestion
To test your locally built SDK, run `eng\dogfood.cmd` after building. That script starts a new Powershell with the environment configured to redirect SDK resolution to your build. Then run `& devenv.exe`.
```

Comment (drewnoakes) [reply README.md]: Thanks!

Comment (drewnoakes) [reply README.md]: Nice. Updated.

**Comment by drewnoakes** [comment]
I didn't mean to create the branch in this repo. Editing the file on github.com seems to have done that automatically (or I missed the option to do it on my fork.)

**Comment by nguerrera** [comment]
No worries, I deleted the branch on merge.

**Pull request merged**